### PR TITLE
add initial list of resourcres

### PR DESCRIPTION
**Pull request by NTCoding**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,38 +10,112 @@ This repository contains a collection of blog posts, videos and other resources
...
+- [Domain-Driven Design Reference](https://domainlanguage.com/wp-content/uploads/2016/05/DDD_Reference_2015-03.pdf), Eric Evans
+
+- [The Anatomy of Domain-Driven Design](https://leanpub.com/theanatomyofdomain-drivendesign), Scott Millett and Samuel Knight
+
+- [DDD: The First 15 Years](https://leanpub.com/ddd_first_15_years), Various Authors
+
+
 ### Written Articles
 
+- [Domain-Driven Design](https://www.thoughtworks.com/insights/blog/domain-driven-design-neednt-be-hard-heres-how-start), Martin Fowler
```
Comment (trondhjort) [inline README.md]: Wrong link? Same as the one under.

**Review by trondhjort — COMMENTED** [review]
Looks good. Hoped there would be room for these:
http://udidahan.com/2009/06/29/dont-create-aggregate-roots/
https://danielwhittaker.me/2020/01/22/are-you-making-these-10-ddd-mistakes/
https://vimeo.com/44235655 (this one made a quite a few things click for be back when)
https://lostechies.com/jimmybogard/2016/06/13/10-lessons-from-a-long-running-ddd-project-part-1/

**Comment by trondhjort** [comment]
Think there is a lot of great stuff in Udi's blog; http://udidahan.com/category/ddd/

**Comment by skleanthous** [comment]
A _lot_ of good resources there, and I very much like it overall 👍 

One addition: while there are a decent amount of popular and approachable resources around decomposing domains, and allocation of responsibilities to them, those (naturally) don't cover ownership of data and how to handle use of shared data across aggregates and services. This is a subject that (at least from my experience) that causes a lot of confusion (especially to beginners of DDD), and also is the source of a lot of the problems they later face through tight / increased coupling, and consistency problems. It is due to the previous that I'd suggest adding one of:

- https://www.youtube.com/watch?v=KkzvQSuYd5I
- https://www.youtube.com/watch?v=jdliXz70NtM

The above two talks are similar, but both are very good, with Adam Ralp's being a bit more on the technical side.

**Comment by cakper** [comment]
One more ebook that we can include: https://www.infoq.com/minibooks/domain-driven-design-quickly/

Other than that I thnink we should decide how to address comments, merge it and let people send PRs :)

**Pull request merged**